FluidNC/src/ProcessSettings.cpp

```diff
@@ -818,7 +819,50 @@ static Error showHeap(const char* value, WebUI::AuthenticationLevel auth_level,
     log_info("Heap free: " << xPortGetFreeHeapSize() << " min: " << heapLowWater);
     return Error::Ok;
 }
-
+//Maslow-specific user functions
```

Comment:
Nice! It's great to have these integrated instead of hacking them into the existing buttons